spec/person_spec.rb

```diff
@@ -0,0 +1,23 @@
+require './lib/person'
+
+describe Person do
+  books = [{ title: 'Managment101', author: 'Rich' }, { title: 'Cooking101', author: 'Fat'}]
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)